plugins/aws/cloudformation/cloudformationInUse.js

```diff
@@ -0,0 +1,49 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'AWS CloudFormation In Use',
+    category: 'CloudFormation',
+    domain: 'Application Integration',
+    description: 'Ensure that Amazon CloudFormation is in use within your AWS account of selected region to automate your infrastructure management and deployment.',
+    more_info: 'AWS CloudFormation is a service that helps you model and set up your AWS resources so that you can spend less time managing those resources and more time focusing on your applications that run in AWS.'+
```

Comment:
```suggestion
    more_info: 'AWS CloudFormation is a service that helps you model and set up your AWS resources so that you can spend less time managing those resources and more time focusing on your applications that run in AWS. '+
```